### ISSUE TITLE
Is it possible to obtain the redirected uri?

### ISSUE THREAD

**Issue by IceDragon200** [excon/excon#527]
Can I obtain the redirected uri (final endpoint) with the `RedirectFollower`?

**Comment by geemus** [comment]
I think it will include the final ip as `remote_ip` on the response, but not the URI. We could potentially add it (I just don't think it has come up before). Could you describe the desired usage a bit so I can make sure we are on the same page? Thanks!

**Comment by IceDragon200** [comment]
@geemus Some requested urls serve no other purpose but to redirect you to another. Like download links (the file was moved, so you want the new path to avoid the redirect page next time), old api routes (the deprecated api route is redirected to the new one), domain name changes (mysite.net > newsite.io)

tl;dr
`url: http://oldsite.net/posts/12` > `redirected_url: https://newsite.net/posts/12`
`url: http://downloads.site.io/?some_query=123456789xyz` > `redirected_url: http://downloads.site.io/files/file_that_you_wanted.zip`

**Comment by geemus** [comment]
@IceDragon200 sorry for the delay. I just did a quick PR that could help with this and is a bit more general/generic. It just makes host/path/port accessible from the response. Since this should already be the final location when the redirect follower succeeds you would be able to just read it off I think. Does that sound reasonable? Any chance you could give it a try and see if it works well for your use case? Thanks!

**Comment by IceDragon200** [comment]
@geemus Yup, should tested it, works nicely, thanks! 
You can close the issue once you merge the pr

**Comment by geemus** [comment]
Merged, thanks!
